Cargo.toml

```
@@ -150,7 +150,7 @@ rusqlite = "0.31"
 rust-embed = "8.6.0"
 scopeguard = { version = "1.2.0" }
 serde = { version = "1.0" }
-serde_json = "1.0"
+serde_json = ">=1.0.97,<1.1"
```

```suggestion
serde_json = "1.0.97"
```

That should be enough. As you can see from [the docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio).

---

Good catch fixed (as a force-push to keep a clean history)